sympy/series/gruntz.py

```diff
@@ -626,9 +626,6 @@ def rewrite(e, Omega, x, wsym):
 
     # Some parts of sympy have difficulty computing series expansions with
     # non-integral exponents. The following heuristic improves the situation:
-    exponent = reduce(ilcm, denominators, 1)
```

Comment:
As you have removed this line, you need to remove `ilcm` from sympy import at the top of the file.